client/Dockerfile.rhel7

```diff
@@ -0,0 +1,20 @@
+FROM rhel7:7.3-released
```

Comment:
This tag definitely doesn't exist in docker hub. Do we need a `FROM` parameter to `osbs-box.py`?